src/library/autodj/autodjprocessor.cpp

```diff
@@ -174,13 +178,21 @@ AutoDJProcessor::AutoDJProcessor(
             ConfigKey(kConfigKey, kTransitionModePreferenceName),
             static_cast<int>(TransitionMode::FullIntroOutro));
     m_transitionMode = static_cast<TransitionMode>(configuredTransitionMode);
+    // TODO(c3n7) Check this out later
+    int configuredFaderMode = m_pConfig->getValue(
+            ConfigKey(kConfigKey, kFaderModePreferenceName),
+            static_cast<int>(FaderMode::Crossfader));
+    m_faderMode = static_cast<FaderMode>(configuredFaderMode);
 }
 
 AutoDJProcessor::~AutoDJProcessor() {
     qDeleteAll(m_decks);
     m_decks.clear();
     delete m_pCOCrossfader;
     delete m_pCOCrossfaderReverse;
+    for (int i = 0; i < 4; i++) {
+        delete m_pVolumeFader[i];
```

Comment:
We try to avoid delete in new code.
Can we patent the the code to the AutoDJProcessor? 
In Mixxx we have a make_patented to make that explicite.

Reply:
I think @daschuer meant "parent"/"make_parented" in case you wondered.

Reply:
`parented_ptr` ;) Prefer to parent all permanent members, then Qt takes care of the deletion.

Reply:
Thank you, I'll do that and push a commit.